portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,52 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+console.log("hello world");
+
+async function getRandomQuoteUsingAsyncAwait() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  document.getElementById('fetchTutorial').innerText = quote;
+}
+
+
+/**
+ * Fetches comments from the servers and adds them to the DOM.
+ */
+function getCommentsold() {
```

Comment:
Can this be deleted now?

Reply:
yes- In terms of workflow, I already fixed these issues in week3step3 but since i started working on this before merging week3step3, it did not get recognized. What should I do in these cases?

Reply:
I would suggest you do the same thing, but then only open PR once the preceding one is merged one master.